content/en/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -44,8 +44,7 @@ and the controller deletes the volume.
 
 ## Owner references, labels, and finalizers {#owners-labels-finalizers}
 
-Like {{<glossary_tooltip text="labels" term_id="label">}}, [owner references]
-(/docs/concepts/overview/working-with-objects/owners-dependents/)
+Like {{<glossary_tooltip text="labels" term_id="label">}}, [owner references](/docs/concepts/overview/working-with-objects/owners-dependents/)
```

Comment:
```suggestion
Like {{<glossary_tooltip text="labels" term_id="label">}},
[owner references](/docs/concepts/overview/working-with-objects/owners-dependents/)
```

Reply:
Hi, do you mean to add a new line break?